packages/frontend/app/lib/api/asset.server.ts

```diff
@@ -24,251 +24,300 @@ import type {
   WithdrawAssetLiquidity,
```

Comment:
This, and other files with the path `/frontend/app/lib/api/*.server.ts`, will have messy diffs but all follow the same change: the query is now nested inside of a try-catch block, and the `catch` block will catch the error, and if it is an `Unauthenticated` Apollo Error, it will throw an error formatted for Remix instead.